xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -5019,7 +5019,7 @@ search.admin.lucene.indexing.action.indexcustom.hqlfilter.title=Same as in searc
 search.admin.lucene.indexing.action.indexcustom.clearindex=Clear the index
 search.admin.lucene.indexing.action.indexcustom.clearindex.title=The index is cleaned before starting to scan database
 search.admin.lucene.indexing.action.indexcustom.onlynew=Only index elements not already indexed
-search.admin.lucene.indexing.action.indexcustom.onlynew.title=A page is loaded and scanned only if its not already in the Lucene index 
+search.admin.lucene.indexing.action.indexcustom.onlynew.title=A page is loaded and scanned only if it is  not already in the Lucene index 
```

Comment:
No, not such a good one. The student has added an extra space which doesn't look good. OIn addition I'd venture that it would be better to write 'if it's not already" rather than "if it is not already". Another possibility is "only if not already in the.." to be shorter/concise.

Reply:
Fixed the extra space which did I noticed initially, but forgot about it https://github.com/xwiki/xwiki-platform/commit/2f943030f8cbf713865a5dcc0979394b99bc664b.